pontoon/contributors/static/css/contributors.css

```diff
@@ -23,16 +23,6 @@ body.top-contributors #heading .legend li .status.fa {
   text-align: right;
 }
 
-.rank {
-  text-align: center;
-}
-
-td.rank {
-  color: #EBEBEB;
-  font-size: 36px;
-  width: 41px;
-}
-
 .contributor {
   display: inline-block;
   position: relative;
```

Comment:
Please set the `.contributor` with to `580px`.

Reply:
Done!

Reply:
Well done! I'm sorry, but we should also increase the `max-width` of `.contributor p` to `520px`.

Reply:
> That doesn't look like it's fixed?
> [#1418 (comment)](https://github.com/mozilla/pontoon/pull/1418#discussion_r334670342)

Just saw it. Done!